app_dart/uuid_enhanced/.cirrus.yml

```diff
@@ -0,0 +1,13 @@
+container:
```

Comment:
Cocoon has `tests.yaml` where we could add this as a standard Cocoon dart project (would be added to the existing LUCI builder)